string.c

```diff
@@ -10023,16 +10031,20 @@ rb_str_scan(VALUE str, VALUE pat)
 
 /*
  *  call-seq:
- *     str.hex   -> integer
+ *    hex -> integer
  *
- *  Treats leading characters from <i>str</i> as a string of hexadecimal digits
+ *  Interprets the leading substring os +self+ as a string of hexadecimal digits
```

Comment:
I think there's a typo:

```suggestion
 *  Interprets the leading substring of +self+ as a string of hexadecimal digits
```